getting-started/scaling-out.md

```diff
@@ -0,0 +1,128 @@
+# Scaling out using Distributed Hypertables
+
+Hypertables are used to handle large amount of data by chunking it up
+in smaller pieces, allowing operations to execute efficiently. When
+the amount of data grows beyond what you can handle with a single
+machine, you can distribute the data over several machines by using
+*distributed hypertables*.
+
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of partitioning of the hypertable and
+distribute the hypertable partitions on *data nodes*.
+
+![Deployment diagram for distributed hypertables]()
+
+## Working with Data Nodes
+
+Data nodes act as the containers of the hypertable partitions and are
+necessary to be able to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that data nodes are added as objects locally and you should
+already have a running database server on some host. When creating the
+data node, you should provide a (local) name to use when referring to
+the data node as well as the host where the hypertable partition
+should be stored.
+
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com');
+SELECT add_data_node('node2', host => 'dn2.example.com');
+```
+
+Deleting a data node is done in a similar manner.
+
+```sql
+SELECT delete_data_node('node1');
+```
+
+Note that a data node cannot be deleted if it contains data for a
+hypertable. You have to ensure either that there is no data on the
+data node, or that the data is replicated to other nodes.
```

Comment:
Replication is mentioned here, but I don't find that this doc gives a clue how to specify replication. Can it be added to this document for completeness?